filters/openpolicyagent/evaluation.go

```diff
@@ -58,7 +57,7 @@ func (opa *OpenPolicyAgentInstance) Eval(ctx context.Context, req *ext_authz_v3.
 		return nil, err
 	}
 
-	err = envoyauth.Eval(ctx, opa, inputValue, result, rego.DistributedTracingOpts(tracing.Options{opa}))
+	err = envoyauth.Eval(ctx, opa, inputValue, result, rego.DistributedTracingOpts(buildTracingOptions(opa.registry.tracer, opa.bundleName, opa.manager)))
```

Comment:
Shouldn't this use `func (opa *OpenPolicyAgentInstance) DistributedTracing() opatracing.Options`?

Reply:
Yes, that would be simpler. Made that change...